logging/cluster-logging-release-notes.adoc

```diff
@@ -8,20 +8,68 @@ toc::[]
+
+* By default, the data model introduced in OpenShift 4.5 gave logs from different namespaces a single index in common. This change made it harder to see which namespaces produced the most logs. The current release, OpenShift Logging 5.2, adds namespace metrics to the *Logging* dashboard in the {product-title} console. With these metrics, you can see which namespaces produce logs and how many logs each namespace produces for a given timestamp. To see these metrics, open the {product-title} console and click *Monitoring* -> *Logging*.
+(link:https://issues.redhat.com/browse/LOG-1680[LOG-1680])
+
+
+// [id="openshift-logging-5-2-0-deprecated-removed-features"]
+// === Deprecated and removed features
+//
+// [IMPORTANT]
+// ====
+// {FeatureName} is a deprecated feature. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+//
+// For the most recent list of major functionality that has been deprecated or removed within {product-title}, refer to the _Deprecated and removed features_ section of the {product-title} release notes.
+// ====
+//
+// [id="openshift-logging-5-2-0-tbd"]
+// ==== TBD-title
+//
+// TBD-description
+
+[id="openshift-logging-5-2-0-bug-fixes"]
+=== Bug fixes
+
+* In previous releases, if you redeployed a full OpenShift Logging-managed Elasticsearch cluster, it might become stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This issue was caused by new certificates preventing the OpenShift Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, The OpenShift Elasticsearch Operator can scale all the data and non-data nodes down and back up again to load the new certificates. The OpenShift Elasticsearch Operator can reach the new nodes after they load the new certificates.
+(link:https://issues.redhat.com/browse/LOG-1524[LOG-1524])
+
+[id="openshift-logging-5-2-0-known-issues"]
+=== Known issues
+
+* There is currently a known issue: If you forward logs to an external Elasticsearch server and change a configured value in the pipeline secret, such as the username and password, the fluentd forwarder loads the new secret but uses the old value to connect to external Elasticsearch server. This issue happens because the Red Hat OpenShift Logging Operator does not currently monitor secrets for content changes.
+
+Workaround: If you change the secret, you can force the Fluentd pods to redeploy by entering `oc delete pod -l component=fluentd`.
+(link:https://issues.redhat.com/browse/LOG-1652[LOG-1652])
```

Comment:
```suggestion
* If you forward logs to an external Elasticsearch server and then change a configured value in the pipeline secret, such as the username and password, the fluentd forwarder loads the new secret but uses the old value to connect to an external Elasticsearch server. This issue happens because the Red Hat OpenShift Logging Operator does not currently monitor secrets for content changes. (link:https://issues.redhat.com/browse/LOG-1652[LOG-1652])
+
As a workaround, if you change the secret, you can force the Fluentd pods to redeploy by entering:
+
[source,terminal]
----
$ oc delete pod -l component=fluentd
----
```

Reply:
Just some suggestions:
I would put the command in its own code block to draw attention to the workaround and make it easier for users to copy/paste to their terminal.

I also think the "There is currently a known issue:" may be redundant here, since the heading already mentions known issues?

Reply:
Thanks. Updated. I was following the conventions established by the Supplementary Style Guide but prefer your version.